### PR TITLE
Update data model and installation

### PR DESCRIPTION
**Pull request by mervey45**
-

### CODE REVIEW

File: tutorials/abap-environment-rap100-enhance-data-model/abap-environment-rap100-enhance-data-model.md [hunk 1]
```diff
@@ -220,7 +221,8 @@ Enhance the metadata extension![ddlx icon](adt_ddlx.png) to change the appearanc
     - Element **`BookingFee`** - should not be displayed in the list table.
     - Element **`TotalPrice`** - should not be displayed in the list table.
     - Element **`CurrencyCode`** - should not be explicitly displayed, neither in the list table nor on the object page.
-      >Note: The currency code will be automatically displayed on the UI thanks to `@consumption` annotations specified for the element `CurrencyCode` in the BO projection view.
+       **Note:** The currency code will be automatically displayed on the UI thanks to `@consumption` annotations specified for the element `CurrencyCode` in the BO projection view.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-rap100-enhance-data-model/abap-environment-rap100-enhance-data-model.md:224]: Please do not use the label "Note".

File: tutorials/abap-install-adt/abap-install-adt.md [hunk 2]
```diff
@@ -22,9 +22,12 @@ author_profile: https://github.com/mervey45
       - Apple Mac OS X 10.14 or higher
 - Microsoft VC Runtime:
       - For Windows OS: [Microsoft Visual C++ 2013 (x64)](https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0) for communication with the back-end system is required.
-- Java Runtime:
-      - ADT is validated and tested against OpenJDK and SapMachine JVMs (OpenJDK / [SapMachine](https://sap.github.io/SapMachine/)).
-      - Recent Eclipse packages already include an  AdoptOpenJDK. Any other JRE found on the system is not used. If this is not desired, see note [3035242](https://launchpad.support.sap.com/#/notes/3035242) on how to remove the bundled JRE and use a custom one.
+    >**Note:** Precisely version Visual Studio 2013 (VC++ 12.0) x64 for communication with the back-end system is required.
```
Comment (MichaelCzcz) [inline tutorials/abap-install-adt/abap-install-adt.md:25]: Please do not use the label "Note".

File: tutorials/abap-install-adt/abap-install-adt.md [hunk 3]
```diff
@@ -22,9 +22,12 @@ author_profile: https://github.com/mervey45
...
+-	Java Runtime:
+      -	ADT is validated and tested against Java versions 11 and 17 (Oracle Java and `OpenJDK`).
+      - The latest Eclipse packages are bundled with [`Eclipse Temurin`](https://adoptium.net/), an `OpenJDK` binary distribution provided by the [`Eclipse Adoptium`](https://projects.eclipse.org/projects/adoptium) project. Any other JRE found on the system is not used. If you want to remove the bundled JRE and use a custom one, see SAP note [3035242](https://launchpad.support.sap.com/#/notes/3035242).
+    >**Note:** No manual installation of an JRE / JDK is required, since it's already bundled with the latest Eclipse packages which are downloadable as a ready-to-run zip-file.
```
Comment (MichaelCzcz) [inline tutorials/abap-install-adt/abap-install-adt.md:29]: Please do not use the label "Note".

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by mervey45** [comment]
@MichaelCzcz I updated the note label

**Pull request merged**